code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Offer.java

```diff
@@ -283,6 +283,17 @@ public void displayed() {
         OptimizeUtils.trackWithData(generateDisplayInteractionXdm());
     }
 
+    /**
+     * Dispatches an event for the Edge network extension to send an Experience Event to the Edge
+     * network with the display interaction data for the given {@code OptimizeProposition} list of
+     * offers.
+     *
+     * @see OptimizeUtils#trackWithData(Map)
+     */
+    public void displayed(List<Offer> offers) {
```

Comment:
Please make this static since its not necessary to call it on an instance of the Offer object

Reply:
Alternatively you can add it in Optimize.java